### PR TITLE
Fix noted item prices & updated hover text [Loot Tracker]

### PR DESCRIPTION
**Pull request by TheStonedTurtle**
Added `getUnnotedItemComposition` to `ItemManager.java`.

Updated Hover Text Format: 

A different take on #4630.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java [hunk 1]
```diff
@@ -113,17 +113,20 @@ private String buildToolTip(ItemManager itemManager, ItemStack item)
 	{
 		final String name = itemManager.getItemComposition(item.getId()).getName();
 		final int quantity = item.getQuantity();
-		final long price = calculatePrice(itemManager, new ItemStack[]{item});
+		ItemPrice p = itemManager.getItemPrice(itemManager.getUnnotedItemComposition(item.getId()).getId());
+		int price = (item.getId() == ItemID.COINS_995 ? 1 : p == null ? 0 : p.getPrice());
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java]:
```
int price = item.getID() == ItemID.COINS_995 ? 1 : (p == null ? 0 : p.getPrice());
```
is much clearer here imo.

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java [hunk 2]
```diff
@@ -113,17 +113,20 @@ private String buildToolTip(ItemManager itemManager, ItemStack item)
...
 
-		return name + " x " + quantity + " (" + price + ")";
+		return "<html>" + name + " x " + quantity + "<br/>"
+				+ "Price: " + StackFormatter.formatNumber(price) + "<br/>"
+				+ "Total: " + StackFormatter.formatNumber(price * item.getQuantity()) + "</html>";
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java]: Use `quantity` instead of `item.getQuantity()`.

**Comment by Nightfirecat** [comment]
I'm not a huge fan of the 3-line format, but the code looks fine.

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java [hunk 3]
```diff
@@ -113,17 +113,20 @@ private String buildToolTip(ItemManager itemManager, ItemStack item)
 	{
 		final String name = itemManager.getItemComposition(item.getId()).getName();
 		final int quantity = item.getQuantity();
-		final long price = calculatePrice(itemManager, new ItemStack[]{item});
+		ItemPrice p = itemManager.getItemPrice(itemManager.getUnnotedItemComposition(item.getId()).getId());
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java]: You are getting itemComposition while not being at clientThread, this is potentionally dangerous

**Pull request closed without merging**